shell/platform/darwin/macos/framework/Headers/FlutterPlatformViews.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERPLATFORMVIEWS_H_
+#define FLUTTER_FLUTTERPLATFORMVIEWS_H_
+
+#import <AppKit/AppKit.h>
+
+#import "FlutterCodecs.h"
+#import "FlutterMacros.h"
+
+@protocol FlutterPlatformViewFactory <NSObject>
+
+/**
+ * Create a Platform View which is an `NSView`.
+ *
+ * Implemented by MacOS plugin code to return an `NSView` to be inserted into the Flutter view
+ * hierarchy.
+ *
+ * The implementation of this method should create a new `NSView` and return it.
```

Comment:
nits:
Since we already mentioned "return", maybe we only need to say "create" here:
```
The implementation of this method should create a new instance of `NSView`.
```

Reply:
Done!